fix: Customization Custom Action (#653)
ZupIT/beagle-docs: content/en/ios/customization/_index.md

---
title: Customization
weight: 139
description: >-

>   You will find here the main classes that are part of Beagle's framework for

Reviewer: ```suggestion
  This section lists the main classes that are part of Beagle's framework for iOS
```

  iOS.
---

---

> These are Beagle for iOS classes:

Reviewer: ```suggestion
These are the iOS classes for Beagle
```
